presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/AgentFileUtils.java

```diff
@@ -36,6 +34,8 @@
 import java.util.stream.Stream;
 
 import static com.google.common.base.Preconditions.checkArgument;
+import static java.io.File.createTempFile;
+import static java.nio.file.Files.copy;
 import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
 import static java.util.stream.Collectors.collectingAndThen;
```

Comment:
It's not functionally part of this pull request, but it'd be nice if the use of `collectingAndThen` were replaced by `ImmutableList.toImmutableList()`.